__tests__/schedule-test.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+jest
+  .dontMock('../schedule.js')
+  .dontMock('minimatch')
+  .dontMock('node-schedule');
+
+require.requireActual('babel-polyfill');
+
+var schedule = require('../schedule.js');
+var currentTime;
+
+describe('Schedule', function() {
+
+  describe('#parse', function() {
+    beforeEach(function(){
+      currentTime = new Date()
+    });
+
+    it('Parses "3 days" as 3 days from now', function() {
+      var parsed: Date = schedule.parse('3 days');
+      var future: Date = new Date();
+      future.setDate(currentTime.getDate() + 3);
+      future.setSeconds(0);
+      future.setMilliseconds(0);
+      expect(parsed.toDateString()).toEqual(future.toDateString())
+    });
+
+    it('Parses "1 hour" as 1 hour from now', function() {
+      var parsed: Date = schedule.parse('1 hour');
+      var future: Date = new Date();
+      future.setHours(currentTime.getHours() + 1);
+      future.setSeconds(0);
+      future.setMilliseconds(0);
+      expect(parsed.toDateString()).toEqual(future.toDateString())
```

Comment:
could you put ; at the end of every single line, not omitting the last one? Thanks!